geomstats/geometry/discrete_curves.py

```diff
@@ -2,8 +2,11 @@
 
 import math
 
+import numpy as np
```

Comment:
Why the numpy import? Do you need anything from the backends?

Reply:
In the worst case, let's still import gs. instead of np. but only run the tests with numpy (and/or autograd) backends